dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.h

```diff
@@ -0,0 +1,17 @@
+﻿#pragma once
+
+#include <FrameworkUdk/PushNotificationsRT.h>
+
+class NotificationListener : public Microsoft::WRL::RuntimeClass<::ABI::Microsoft::Internal::PushNotifications::INotificationListener>
```

Comment:
A NotificationListener is not a per app component. It is supposed to cover all apps. So this tells me that we have to rename these 2 components:
NotificationListener -> AppListener?
NotificationListenerManager -> NotificationListener (This is the real listener IMHO)